R/install_easystats.R

```diff
@@ -5,10 +5,12 @@
 #' versions from CRAN. If the development versions are installed, packages
 #' will be installed from the stable branch (master/main) for each package.
 #'
-#' @param source Character. Either `"development"` or `"cran"`. If `"cran"`,
-#'   packages will be installed from the default CRAN mirror returned by
-#'   `getOption("repos")['CRAN']`. If `"development"` (the default), packages
-#'   are installed from the r-universe repository (<https://easystats.r-universe.dev/>).
+#' @param source Character. Either `"development"`, `"cran"` or `"github"`. If
+#'   `"cran"`, packages will be installed from the default CRAN mirror returned
+#'   by `getOption("repos")['CRAN']`. If `"development"` (the default), packages
+#'   are installed from the r-universe repository
+#'   (<https://easystats.r-universe.dev/>). `"github"` installs the latest
+#'   version from the GitHub-repositories main-branch.
```

Comment:
We should emphasize that `source = "github"` is only interesting if r-universe is down. In other scenarios, if the user wants the latest versions, then R-universe since it provides binaries.

Reply:
Yes, will add this to the docs